libs/Transisthor/Transisthor.cpp

```diff
@@ -353,6 +354,13 @@ void Transisthor::componentConvertVelocityType(unsigned short id, void *byteCode
 
     _ecsWorld.updateComponentOfAnEntityFromGivenDistinctiveComponent<Networkable, Velocity>(
         Networkable(id), newComponent);
+    std::vector<std::shared_ptr<ecs::Entity>> joined = _ecsWorld.joinEntities<Player>();
+    for (auto it : joined) {
+        if (it->getComponent<Networkable>().id == id) {
+            it->getComponent<AfkFrequency>().frequency = it->getComponent<AfkFrequency>().baseFrequency;
```

Comment:
?????????????

Reply:
Refering to the PR description : "Update the last interaction time for -> Shooting, Moving" -> When i receive a Velocity component, i will check if it's one of a player. And if it's a velocity of a player, i will update the last interaction time :)